docs/howto/manage-charms.rst

```diff
@@ -250,6 +255,26 @@ that at present this must be a Discourse page. E.g.,
 
     See more: :ref:`charmcraft-yaml-key-links`
 
+Smaller charms can have a single-page documentation in the Description tab. 
+Bigger charms with multi-page documentation, should have a full
+[Diátaxis](https://diataxis.fr/) navigation tree in the Description tab, 
+or just provide a brief description with a link to an external documentation set. 
```

Comment:
```suggestion
A smaller charm can have a single-page documentation for its description. 
A bigger charm, with multi-page documentation, should have a full
[Diátaxis](https://diataxis.fr/) navigation tree in its description, 
or just provide a brief description with a link to an external documentation set. 
```

I think we can reduce how much we speak about the tab itself.

Reply:
I don't think we can replace all mentions of the tab, since we need to refer to the exact UI element sometimes, but I've adjusted the text to minimise the usage. Up to one per paragraph.

Reply:
Looks great!